lib/madwf_tensor.cu

```diff
@@ -0,0 +1,78 @@
+#include <color_spinor_field.h>
+#include <tunable_nd.h>
+#include <device_vector.h>
+#include <madwf_transfer.h>
+#include <kernels/madwf_tensor.cuh>
+#include <tunable_reduction.h>
+#include <madwf_ml.h>
+
+namespace quda
+{
+  namespace madwf_ml
+  {
+
+    template <class storage_t> class tensor_5D_wrapper : public TunableMultiReduction<1>
+    {
+      const ColorSpinorField &x;
+      const ColorSpinorField &y;
+      using Arg = Tensor5DReduceArg<storage_t>;
+      using reduce_t = typename Arg::reduce_t;
+      reduce_t *wm_p;
+
+    private:
```

Comment:
This private declaration is redundant in a class.

Reply:
https://github.com/lattice/quda/pull/1061/commits/703f9d363082c78607d77ac0d576e66733264f35